source/geometries/Next100EnergyPlane.cc

```diff
@@ -46,24 +46,24 @@ namespace nexus {
     // hole_lat1_posy_ (-203.8 * mm),
     // hole_lat2_posx_ (-415 * mm),
     // hole_lat2_posy_ (-311.2 * mm),
-    hut_int_diam_ (76. * mm),
-    hut_thickn_ (5. * mm),
+    hut_int_diam_ (64. * mm),
+    hut_thickn_ (6. * mm),
```

Comment:
This variable is only used once in the whole class. Perhaps we should define hut_diam_(line 152) as global (which is used more) and whose value is explicitly indicated in the drawings.

Reply:
Done